### ISSUE TITLE
Strict mode warns over custom font tokens

### ISSUE THREAD

**Issue by raybotha** [GeekyAnts/NativeBase#4111]
**Describe the bug**
Using a custom font with strict mode causes the following error:

<img width="372" alt="Screen Shot 2021-09-23 at 23 01 10" src="https://user-images.githubusercontent.com/9902896/134583422-8dabd331-7aa8-44de-a97b-00e82e10987c.png">

**To Reproduce**
Steps to reproduce the behaviour:

Text example:

```JSX
<Text fontWeight="800">Hello world</Text>
```

Theme config:

```JSX
  fontConfig: {
    Lato: {
      100: {
        normal: "Lato-Hairline",
        italic: "Lato-Hairline-Italic",
      },
      200: {
        normal: "Lato-Thin",
        italic: "Lato-Thin-Italic",
      },
      300: {
        normal: "Lato-Light",
        italic: "Lato-Light-Italic",
      },
      400: {
        normal: "Lato-Regular",
        italic: "Lato-Italic",
      },
      500: {
        normal: "Lato-Medium",
        italic: "Lato-Medium-Italic",
      },
      600: {
        normal: "Lato-Semibold",
        italic: "Lato-Semibold-Italic",
      },
      700: {
        normal: "Lato-Bold",
        italic: "Lato-Bold-Italic",
      },
      800: {
        normal: "Lato-Heavy",
        italic: "Lato-Heavy-Italic",
      },
      900: {
        normal: "Lato-Black",
        italic: "Lato-Black-Italic",
      },
    },
  },
  fonts: {
    heading: "Lato",
    body: "Lato",
    mono: "Lato",
  }
```

Changing `Lato-Heavy` to `XYZ` in the theme causes the warning to appear for `XYZ` instead.

**Expected behaviour**
No warnings.

**Platform**

- NativeBase v3.2.0
- iPhone 8, Pixel 5
- iOS 15, Android 11
- React Native v0.65.1
- node v16.10.0
- yarn v3.0.1

**Comment by sgolubovic** [comment]
@raybotha Same issue, my workaround was to add all font values from **fontConfig** to **fonts**
`
...
fonts: {
  heading: "Lato",
  body: "Lato",
  mono: "Lato",
  Lato-Heavy: "Lato"
  ...(other fonts)
}
`

**Comment by raybotha** [comment]
Issue not resolved AFAIK

**Comment by dkulakov** [comment]
Have the same bug

**Comment by surajahmed** [comment]
We're looking into this. Hope to fix this in the coming release. Thanks for reporting!

**Comment by dkulakov** [comment]
Looks like this bug was fixed into `"native-base": "3.3.5"`

**Comment by surajahmed** [comment]
This issue is fixed in version `3.3.5`.
